vmm/src/memory_manager.rs

```diff
@@ -1081,9 +1120,14 @@ impl MemoryManager {
             arch_mem_regions,
             ram_allocator,
             dynamic,
+            #[cfg(target_arch = "aarch64")]
+            uefi_flash: None,
         };
 
         memory_manager.allocate_address_space()?;
+        #[cfg(target_arch = "aarch64")]
```

Comment:
nit: I think there should be line of whitespace above here

Reply:
Sure.